### PR TITLE
Update doc site quickstart page to reflect v2.0 API

### PR DESCRIPTION
**Pull request by weijinglok**
This PR updates the doc's site quickstart page to reflect the v2.0 API.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cleanlab documentation
 ======================
 
-`cleanlab <https://github.com/cleanlab/cleanlab>`_ **automatically finds and fixes errors in your ML datasets.**
+`cleanlab <https://github.com/cleanlab/cleanlab>`_ **automatically finds and fixes label issues in your ML datasets.**
 
 | This reduces manual work needed to fix data issues and helps train reliable ML models on noisy real-world datasets. ``cleanlab`` has already found thousands of `label errors <https://labelerrors.com>`_ in ImageNet, MNIST, and other popular ML benchmarking datasets, so let's get started with yours!
```
Comment (jwmueller) [inline docs/source/index.rst:6]: [Nit] If we go with the above wording, should change this to reduce redundancy:
"to fix data issues"
-> 
"to fix data errors"

Comment (weijinglok) [reply docs/source/index.rst:6]: i've actually changed the above wording to "label issues" too. do you mean we should change "to fix data issues" to "to fix data label issues"?

File: docs/source/index.rst [hunk 2]
```diff
@@ -35,29 +35,29 @@ Quickstart
...
 
 ..
-   todo - include the url for tf and torch beginner tutorials
+   TODO - include the url for tf and torch beginner tutorials
 
 3. Train robust models with noisy labels using ``LearningWithNoisyLabels``.
 ---------------------------------------------------------------------------
 
-``cleanlab``'s ``LearningWithNoisyLabels`` adapts any classification model, ``clf``, to a more reliable one by allowing it to train directly on partially mislabeled datasets. 
+``cleanlab``'s ``LearningWithNoisyLabels`` adapts any scikit-learn compatible classification model, ``clf``, to a more reliable one by allowing it to train directly on partially mislabeled datasets.
```
Comment (jwmueller) [inline docs/source/index.rst:60]: [Nit] "adapts any scikit-learn compatible classification model"
-> 
"adapts any existing (scikit-learn compatible) classification model"

Comment (weijinglok) [reply docs/source/index.rst]: done

File: docs/source/index.rst [hunk 3]
```diff
@@ -35,29 +35,29 @@ Quickstart
...
-   todo - include the url for tf and torch beginner tutorials
+   TODO - include the url for tf and torch beginner tutorials
 
 3. Train robust models with noisy labels using ``LearningWithNoisyLabels``.
 ---------------------------------------------------------------------------
 
-``cleanlab``'s ``LearningWithNoisyLabels`` adapts any classification model, ``clf``, to a more reliable one by allowing it to train directly on partially mislabeled datasets. 
+``cleanlab``'s ``LearningWithNoisyLabels`` adapts any existing (scikit-learn compatible) classification model, ``clf``, to a more reliable one by allowing it to train directly on partially mislabeled datasets. 
 
 When the ``.fit()`` method is called, it automatically identifies and removes any examples that are deemed "noisy" in the provided dataset before returning a final trained model.
```
Comment (jwmueller) [inline docs/source/index.rst:62]: "it automatically identifies and removes any examples that are deemed "noisy" in the provided dataset before returning a final trained model."
-> 
"it automatically removes any examples that are identified as "noisy" in the provided dataset and returns a model  trained on only the clean data.

File: docs/source/index.rst (hunk 1 above)
Comment (jwmueller) [reply docs/source/index.rst]: No I am saying: Keep above as is.

Change: "to fix data issues"
->
"to fix data errors"

Comment (weijinglok) [reply docs/source/index.rst]: done!

File: docs/source/index.rst (hunk 3 above)
Comment (weijinglok) [reply docs/source/index.rst]: done!

**Review by jwmueller — APPROVED** [review]
LGTM

**Pull request merged**